packages/common/utils/src/groupBy.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('groupBy 함수는', () => {
     { groupName: '부엉이', value: 6 },
   ];
 
-  it('createKey의 return value를 이용해서 item을 묶는다', () => {
+  it('groups items using the return value of createKey', () => {
```

Comment:
I think `it groups` is wrong gramatically
```suggestion
  it('should group items using the return value of createKey', () => {
```

you can refer https://github.com/toss/slash/pull/267#discussion_r1248051773

Reply:
Thank you for letting me know 👍

[chore: correct the test descriptions grammatically.](https://github.com/toss/slash/pull/360/commits/b6d7a8ce28e88d3194eef881977d3942911b3945)